docs/development.md

```diff
@@ -321,3 +321,30 @@ Inspired from: [github.com/kubernetes/community/contributors/devel/api-conventio
   - Add an example in docs/example with its explanation README, for e.g. [health](https://github.com/kedgeproject/kedge/tree/master/docs/examples/health).
   - Add an e2e test on above example, for e.g. see test code for [health](https://github.com/kedgeproject/kedge/blob/cfee15ffde02c611d08420699a43869706be2d53/tests/e2e/e2e_test.go#L272).
   - Add this feature information to file-reference, for e.g. see [health section](https://github.com/kedgeproject/kedge/blob/master/docs/file-reference.md#health).
+
+### golang dependency import conventions
+
+imports MUST be arranged in three sections, separated by empty line.
+
+```go
+stdlib
+kedge
+thirdparty
+```
+
+for e.g.
+
+```go
+    "fmt"
+    "io"
+    "os/exec"
+
+    pkgcmd "github.com/kedgeproject/kedge/pkg/cmd"
+    "github.com/kedgeproject/kedge/pkg/spec"
+
+    "github.com/ghodss/yaml"
+    "github.com/pkg/errors"
+```
+
+And once arranged in such sections let go tooling take care of sorting the
```

Comment:
Once arranged, let `gofmt` sort the sequence of imports.

Would be a better sentence

Reply:
done